ansible/playbooks/roles/common/tasks/configure_apt.yml

```diff
@@ -6,12 +6,13 @@
     owner: root
     group: root
     mode: 0644
-  register: apt_sources
+  notify:
+  - update apt cache
 
-- name: "run apt update if we have modified the sources"
-  apt:
-    update_cache: true
-  when: apt_sources.changed
+- name: cleanup old backports config
+  apt_repository:
+    repo: "deb http://deb.debian.org/debian {{ ansible_distribution_release }}-backports main contrib non-free"
```

Comment:
I think this will remove current backports as it depends on ansible variable, but I am not sure.

Reply:
I'm not sure what you mean, removing is what I'm intending to do here.

Reply:
But you are also adding it in https://github.com/FOSDEM/infrastructure/pull/258/files#diff-ea8a784d3451dd6923172e130674fc1aR3. Are you sure there won't be conflict?

Reply:
This task is `state: absent` to remove it so the new line is not a conflict.